plugins/techdocs-backend/src/techdocs/stages/generate/techdocs.ts

```diff
@@ -46,17 +48,29 @@ export class TechdocsGenerator implements GeneratorBase {
     );
 
     try {
-      await runDockerContainer({
-        imageName: 'spotify/techdocs',
-        args: ['build', '-d', '/result'],
-        logStream,
-        docsDir: directory,
-        resultDir,
-        dockerClient,
-      });
-      this.logger.info(
-        `[TechDocs]: Successfully generated docs from ${directory} into ${resultDir}`,
-      );
+      const mkdocsInstalled = await commandExists('mkdocs');
```

Comment:
I'm thinking this could be a more explicit switch, either through config or in code inside the `backend` package. Basically passing in the runner you want to use to the `techdocs` router

Reply:
Yea, one way to do it could even be to create it as it's own generator and just provide that in the backend setup of techdocs instead of this one. That would make it more explicit that you really want to run the local mkdocs.